archivebox/extractors/__init__.py

```diff
@@ -162,7 +162,7 @@ def archive_link(link: Link, overwrite: bool=False, methods: Optional[Iterable[s
 
         write_link_details(link, out_dir=out_dir, skip_sql_index=False)
 
-        log_link_archiving_finished(link, link.link_dir, is_new, stats, start_ts)
+        log_link_archiving_finished(link, out_dir, is_new, stats, start_ts)
```

Comment:
`out_dir` should already be the custom output dir or the link's archive dir:
https://github.com/ArchiveBox/ArchiveBox/blob/32eef51081e9858b6fd2ae4bcb2ad47186d8b14d/archivebox/extractors/__init__.py#L90